test/functional/schema-builder/change-column.ts

```diff
@@ -22,6 +22,42 @@ describe("schema builder > change column", () => {
     beforeEach(() => reloadTestingDatabases(connections))
     after(() => closeTestingConnections(connections))
 
+    it("uses ALTER COLUMN when increasing varchar length (postgres)", () =>
```

Comment:
Postgres shouldn't be mentioned here; this test should work for all databases supporting this feature.

It would be worth adding another test for reducing the column length as well (I suspect this might fail on some databases).